recipes/natural_language_processing/chatbot/provision/playbook.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Test Environment Provisioning
+  hosts: test_environments
+  remote_user: fedora
+  become: true
+  gather_facts: false
+  vars:
+    arch: "amd64"
+
+  tasks:
+  
+  - name: Wait until the instance is ready
+    ansible.builtin.wait_for_connection:
+    delay: 10
+    timeout: 60
+  
+  - name: Gather facts for first time
+    ansible.builtin.setup:
+
+  - name: Set variable arch
+    set_fact:
+      arch: "arm64"
+    when: ansible_architecture == "aarch64"
+
+  - name: Download GPG key file
+    ansible.builtin.get_url:
+      url: "https://s3.eu-west-2.amazonaws.com/amazon-ssm-region/latest/linux_{{ arch }}/amazon-ssm-agent.rpm.sig"
+      dest: "/tmp/gpg_key.pub"
+
+  - name: Import GPG key
+    ansible.builtin.rpm_key:
+      state: present
+      key: "/tmp/gpg_key.pub"
+
+  - name: Required packages
+    ansible.builtin.package:
+      name: "https://s3.eu-west-2.amazonaws.com/amazon-ssm-us-east-2/latest/linux_{{ arch }}/amazon-ssm-agent.rpm"
```

Comment:
Why?  This thing has deep hooks into the OS...

Reply:
It is needed for unlocking the session manager. It will be removed in the future.

Reply:
The future has arrived. Removed :-)